Lib/tkinter/ttk.py

```diff
@@ -570,6 +571,15 @@ def state(self, statespec=None):
         return self.tk.splitlist(str(self.tk.call(self._w, "state", statespec)))
 
 
+    def cget(self, key):
+        """Return the resource value for a KEY given as string."""
+        if isinstance(return_value := super().cget(key), _tkinter.Tcl_Obj):
+            return str(return_value)
```

Comment:
```suggestion
		return_value = super().cget(key)
        if isinstance(return_value, _tkinter.Tcl_Obj):
            return str(return_value)
```

For readability I would split it like that